### DISCUSSION TITLE
Ability to run the software without internet.

### DISCUSSION THREAD

**Discussion by Uilsonjw** [sircharlo/meeting-media-manager#233] [Ideas] [answered]
We enjoyed a lot the option to play the medias on a second monitor in special during the hybrid meetings.

Once we can decide to use JW Media Fetcher to play the medias stead JW Library, wold be good if the JW Media Fetcher has the ability to run without internet. Otherwise, if the internet connection fails we will not be able to play the medias during the meeting.

**Comment by sircharlo** [comment]
Very true. JWMMF started when meetings were on Zoom, therefore a meeting with no internet wasn't really possible! Now things are changing. I'll update the logic flow to allow for situations where the internet is down but we still need to show media. Thanks for the idea!

**Comment by sircharlo** [comment, marked as answer]
Implemented in https://github.com/sircharlo/jw-meeting-media-fetcher/commit/7c87139872252eef1553841d0633f1210010749f. Will be released soon.
